components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/services/TenantMgtImpl.java

```diff
@@ -563,4 +590,32 @@ private Tenant getTenantFromTenantManager(String tenantUniqueID) throws TenantMa
         }
         return tenant;
     }
+
+    private static TenantInfoBean buildTenantInfoBean(Tenant tenant) {
+
+        TenantInfoBean bean = new TenantInfoBean();
+        if (tenant != null) {
```

Comment:
Null check seems to be not adding much value here. Just before this method is called, object is already accessed and could have caused a NPE already.

Reply:
remove the redundant null check